src/SFML/Window/Win32/VulkanImplWin32.cpp

```diff
@@ -35,72 +36,51 @@
 #include <vulkan.h>
 #include <string>
 #include <map>
+#include <memory>
 #include <cstring>
 
 
 namespace
 {
     struct VulkanLibraryWrapper
     {
-        VulkanLibraryWrapper() :
-        library(nullptr)
-        {
-        }
-
         ~VulkanLibraryWrapper()
         {
-            if (library)
-                FreeLibrary(library);
+            if (libraryPtr)
+                libraryPtr.release();
```

Comment:
This is a memory leak. "Release" does not mean "free memory". It releases ownership so that another code has to free it. Unique pointers don't have to be freed manually. That's the whole point

Reply:
Yep, that's mistake. It should be `reset()`.

Reply:
For the record, the reset will be called automatically on destruction too.

Reply:
Indeed. It's totally pointless to call anything here. I'll just remove destructor.
(well except if we want to leak some memory 🙂 )